### ISSUE TITLE
release a new patch for the django-seo2

### ISSUE THREAD

**Issue by abhyudaypratap** [romansalin/django-seo2#20]
Update the version on python package index.
Version on pypi is 12 commits behind master.

**Comment by mic159** [comment]
Yes, please release a new version!

**Comment by mic159** [comment]
bump!

**Comment by qurm** [comment]
Yes, agree - and good to confirm that it works with Django 1.10.  I believe it does.

**Comment by mic159** [comment]
Looks like he changed companies.
Maybe you can contact him there.

https://github.com/Nitrino/ex_aviasales/blob/28463840a408bee53a579298eee1c66086dc84cf/src/app/employees/employees.json#L82
